docs/server/source/production-deployment-template/node-on-kubernetes.rst

```diff
@@ -488,6 +488,27 @@ You can check its status using: ``kubectl get pvc -w``
 Initially, the status of persistent volume claims might be "Pending"
 but it should become "Bound" fairly quickly.
 
+.. Note::
+    The default Reclaim Policy for dynamically created persistent volumes is ``Delete``
+    which means the PV and its associated Azure storage resource will be automatically
+    deleted on deletion of PVC or PV. In order to prevent this from happening do
```

Comment:
Just keep PVC, because whatever the reclaim policy if you go ahead and delete the PV, it will be deleted. please, correct me if I am wrong.

Reply:
if the reclaim policy is retain the Storage disk is not deleted even if you delete the PV